zebrad/src/config.rs

```diff
@@ -53,3 +53,16 @@ impl Default for MetricsSection {
         }
     }
 }
+
+#[cfg(test)]
+mod test {
+    #[test]
+    fn test_toml_ser() -> color_eyre::Result<()> {
+        let default_config = super::ZebradConfig::default();
+        println!("Default config: {:?}", default_config);
+
+        println!("Toml:\n{}", toml::Value::try_from(&default_config)?);
```

Comment:
as a follow-on issue: we might consider adding a convenience method to the config type like `toml_string` or something that does this incantation? not sure if that's a good solution (certainly has downsides relative to plain serde) but i suspect this won't be the last time this issue bites us

Reply:
How about instead of `toml_string` we just shove the `Value::try_from` stuff into a `fmt::Display` impl on the Config type?